### ISSUE TITLE
PP-OCRv5-mobile-det 检测模型训练多轮之后 precision: 0.0, recall: 0.0, hmean: 0 这些还是 0，loss 在 8-9 中间来回跳

### ISSUE THREAD

**Issue by wenming520520520** [PaddlePaddle/PaddleOCR#17923]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

PP-OCRv5-mobile-det 检测模型训练多轮之后 precision: 0.0, recall: 0.0, hmean: 0，loss 在 8-9 中间来回跳，

<img width="3734" height="1086" alt="Image" src="https://github.com/user-attachments/assets/b3947874-9fc5-4e90-857f-74699a25719b" />

不清楚是什么原因引起的 训练集数量 85 左右，想要检测出电表上的电量数字框，导出模型识别不出框 使用的的paddleLabel 标注工具，应用场景是电表识别，数据集已经拆分预训练模型 ，PP-OCRv5-mobile-rec 识别模型，识别效果很好，检测模型找不到框，麻烦大神看下什么原因，哪里出现问题了该怎么优化 https://paddleocr.bj.bcebos.com/pretrained/PPLCNetV3_x0_75_ocr_det.pdparams
 PP-OCRv5-mobile-det  
下面是我的yaml参数配置
Global:
  model_name: PP-OCRv5_mobile_det
  debug: false
  use_gpu: true
  epoch_num: &epoch_num 100         # 小数据集多训几轮，配合 early stop
  log_smooth_window: 10
  print_batch_step: 20
  log_interval: 1
  save_model_dir: /home/aistudio/output/meter_det
  save_epoch_step: 10
  eval_batch_step: [0, 20]           # 从第0轮开始每5轮eval一次
  cal_metric_during_train: false
  checkpoints:
  #checkpoints: /home/aistudio/output/meter_det/best_model/model
  pretrained_model: https://paddleocr.bj.bcebos.com/pretrained/PPLCNetV3_x0_75_ocr_det.pdparams
  save_inference_dir: /home/aistudio/output/meter_det/inference
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt
  d2s_train_image_shape: [3, 640, 640]
  distributed: false
  use_new_export: false
  export_for_lite: true
  use_amp: true

Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: PPLCNetV3
    scale: 0.75
    det: True
  Neck:
    name: RSEFPN
    out_channels: 96
    shortcut: True
  Head:
    name: DBHead
    k: 50                   # 恢复默认 k=50，配合后处理参数调整
    fix_nan: True

Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine              # 改为 Cosine，小数据集收敛更稳
    learning_rate: 0.0005
    warmup_epoch: 2
    total_epoch: *epoch_num
  regularizer:
    name: L2
    factor: 5.0e-05

PostProcess:
  name: DBPostProcess
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5

Metric:
  name: DetMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: /home/aistudio/data/datasets/det/
    label_file_list:
      - /home/aistudio/data/datasets/det/train.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null      # 先编码原始坐标
    - CopyPaste: null
    - IaaAugment:
        augmenter_args:
        - type: Affine
          args:
            rotate:
            - -5
            - 5
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    # 关键：大图随机裁，裁剪尺寸要够大保证裁到数字区域
    - EastRandomCropData:
        size:
        - 960
        - 960
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4        # 恢复标准收缩比，小目标用0.2会让GT消失
        thresh_min: 0.3
        thresh_max: 0.7
        #total_epoch: *epoch_num
    - MakeShrinkMap:
        shrink_ratio: 0.4        # 同上
        min_text_size: 8         # 允许最小4px文本
        #total_epoch: *epoch_num
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - threshold_map
        - threshold_mask
        - shrink_map
        - shrink_mask
  loader:
    shuffle: true
    drop_last: true
    batch_size_per_card: 8   
    num_workers: 0           

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: /home/aistudio/data/datasets/det/
    label_file_list:
      - /home/aistudio/data/datasets/det/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - DetResizeForTest:
        limit_side_len: 640    # 验证也保持高分辨率
        limit_type: max
    - NormalizeImage:
        scale: 1./255.
        mean:
        - 0.485
        - 0.456
        - 0.406
        std:
        - 0.229
        - 0.224
        - 0.225
        order: hwc
    - ToCHWImage: null
    - KeepKeys:
        keep_keys:
        - image
        - shape
        - polys
        - ignore_tags
  loader:
    shuffle: false              # 验证集不需要打乱
    drop_last: false
    batch_size_per_card: 1
    num_workers: 2

profiler_options: null
训练数据目录
<img width="584" height="246" alt="Image" src="https://github.com/user-attachments/assets/633345d7-241d-41ad-aa9c-e6151adff366" />
训练数据内容
<img width="1332" height="914" alt="Image" src="https://github.com/user-attachments/assets/0afe9e37-4ebc-42d5-aa4f-c7f042e3d622" />
训练照片举例

<img width="1972" height="2648" alt="Image" src="https://github.com/user-attachments/assets/e6f85395-82e1-484d-9c2b-c41017e23f7c" />

### 🏃‍♂️ Environment (运行环境)

paddlepaddle-gpu==3.0.0
paddlepaddleOCR=3.3.3
环境 百度飞桨aiStudio GPU 环境

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

<img width="1408" height="330" alt="Image" src="https://github.com/user-attachments/assets/d647b4e3-bf0a-4244-9b95-6b50eef15858" />

**Comment by cuicheng01** [comment]
是否方便share数据集我们复现下呢？

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_

**Comment by DestoryVIP** [comment]
标注的数字太小了，你应该扩大一下标注的图片，顺便把标注也同步扩大一下就行了
